indico/modules/events/registration/templates/management/regform_display.html

```diff
@@ -145,6 +153,41 @@
                     title: $this.data('title')
                 });
             });
+
+            {{ forbidden_visibility_forms|tojson }}.forEach(function(regform) {
+                var $item = $(`.js-sortable-list-widget li[data-item-id="${regform}"]`);
+                $item.qtip({
+                    style: {
+                        classes: 'qtip-danger',
+                    },
+                    content: {
+                        prerender: true,
+                        text: $T('Show all participants can only be set if there are no registered users.'),
+                    },
+                    position: {
+                        my: 'left middle',
+                        at: 'right middle',
+                    },
+                    show: '',
+                    hide: '',
+                });
+                $item.addClass('forbidden-visibility');
+            });
+            var invalid_forms = 0;
+            $('.js-sortable-list-widget .show-all').on('sortreceive', function(evt, ui) {
+                if (ui.item.hasClass('forbidden-visibility')) {
+                    ui.item.qtip('show');
+                    invalid_forms++;
+                    $form.find(":submit").attr('disabled', 'disabled');
```

Comment:
`.prop('disabled', true)`

Reply:
and we usually use single quotes *except* when that would cause backslash escaping within the string